src/main/java/ocl/service/TransformationService.java

```diff
@@ -1234,7 +1299,7 @@ private boolean isNb(NodeList nodeList){
      * @throws IOException
      * @throws TransformerException
      */
-    private HashMap<String,Node> addObject(Document doc, NodeList nodeList, String s, boolean begin) throws IOException, TransformerException {
+    private HashMap<String,Node> addObject(Document doc, NodeList nodeList, String s, boolean begin) {
```

Comment:
Does this function have to be kept?

Reply:
Yes! it is still used in the code. Only exceptions were managed differently